src/wizard/capacitor/index.md

```diff
@@ -43,9 +43,13 @@ public class MainActivity extends BridgeActivity {
   public void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     // Initializes the Bridge
-    this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
-      add(SentryCapacitor.class);
-    }});
+    // Capacitor 3
+    registerPlugin(SentryCapacitor.class);
```

Comment:
Just to confirm:
@ludufre said -> https://github.com/getsentry/sentry-docs/pull/5093/files

> With Capacitor 3.0 not need to modify the MainActivity.java (part Android Specifics in tutorial)

But actually, you do need to modify, it's just a different method or `registerPlugin` isn't even needed at all?

Reply:
He's correct, I was actually looking for another Issue https://github.com/getsentry/sentry-capacitor/issues/174

On capacitor 2 you need to initialize it but not on Capacitor 3, Unless you're initializing  it using the deprecated code.

Updated the wizards to point out that only capacitor 2 needs to add the Plugin and also added a troubleshooting if folks are using the deprecated code.